minimesos/src/main/java/com/containersol/minimesos/main/CommandUp.java

```diff
@@ -129,7 +132,7 @@ public void execute() {
         ClusterArchitecture clusterArchitecture = getClusterArchitecture();
 
         startedCluster = new MesosCluster(clusterArchitecture);
-        startedCluster.start(getTimeout());
+        startedCluster.start();
         startedCluster.waitForState(state -> state != null);
```

Comment:
because it's already available in the ClusterConfig and it's has been updated with CLI parameter, if it was provided. `CommandUp#getTimeout()` can return `null`